src/go/k8s/apis/redpanda/v1alpha1/redpanda_clusterspec_types.go

```diff
@@ -112,11 +112,10 @@ type Auth struct {
 
 // SASL is a top level field of the values file
 type SASL struct {
-	Enabled   bool   `json:"enabled"`
-	Mechanism string `json:"mechanism,omitempty"`
-	SecretRef string `json:"secretRef,omitempty"`
-	// +optional
-	Users []*UsersItems `json:"users"`
+	Enabled   bool          `json:"enabled"`
+	Mechanism string        `json:"mechanism,omitempty"`
+	SecretRef string        `json:"secretRef,omitempty"`
```

Comment:
If those strings have `omitempty` json annotation should it be a reference to string?

Reply:
yep!